languages/grammar/fr.json

```diff
@@ -0,0 +1,64 @@
+{
+    "meta": {
+        "regExpFlags": "gi"
+    },
+    "v5": {
+        "article": [
+            ["^ Acc[èe]s ", " l’accès "],
+            ["^ All[ée]e ", " l’allée "],
+            ["^ (L['’])?Autoroute ", " l’autoroute "],
+            ["^ Avenue ", " l’avenue "],
+            ["^ Boulevard ", " la boulevard "],
+            ["^ Chemin ", " la chemin "],
+            ["^ Cours ", " la cours "],
+            ["^ Entr[ée]e ", " l’entrée "],
+            ["^ Impasse ", " l’impasse "],
+            ["^ Parvis ", " la parvis "],
+            ["^ Passage ", " la passage "],
+            ["^ Place ", " la place "],
+            ["^ Petit[\\- ]Pont ", " la petit-pont "],
+            ["^ Pont ", " la pont "],
+            ["^ Promenade ", " la promenade "],
+            ["^ Quai ", " la quai "],
+            ["^ Route ", " la route "],
+            ["^ Rue ", " la rue "],
+            ["^ Sortie ", " la sortie "],
+            ["^ Souterrain ", " la souterrain "],
+            ["^ Square ", " la square "],
+            ["^ Tunnel ", " la tunnel "],
+            ["^ Voie ", " la voie "]
+        ],
+        "elision": [
+            ["^ Le ", "  du "],
+            ["^ Les ", "  des "],
+            ["^ La ", "  de "],
```

Comment:
There is no elision here: « de la » is grammaticaly correct. You only need to get rid of the first capital.